tests/main/install-hook/task.yaml

```diff
@@ -0,0 +1,61 @@
+summary: Check install and remove hooks.
+
+prepare: |
+    echo "Build basic test package"
+    snapbuild $TESTSLIB/snaps/snap-install-hooks .
```

Comment:
There's a better helper, `install_local` that caches built snaps and relieves you from the need to remove them below.